package q11317;
public class ExceptionDemo1 {
	public static void main(String[] args) {
		int number = 34;
		int divisor = 2;
		int result = number / divisor;
		System.out.println("result = " + result);
	}
}
